aws/amazon-chime-sdk-js
# Video test app integration test

**Issue #:**

**Description of changes:**
Add end-to-end test for video test app. 
Tests cover apis:
1. addVideoTile 
2. startLocalVideoTile
3. hasStartedLocalVideoTile
4. haveVideoTilesWithStreams
5. haveVideoTileForAttendeeId
6. getLocalVideoTile
7. getAllVideoTiles
8. getAllRemoteVideoTiles 
9. removeAllVideoTiles
10. stopLocalVideoTile 
11. sendTileStateUpdate

Test cover scenarios:
    * 2 attendees join from 2 different browser sessions
    * Both attendees turn on their local video
    * Verify that both attendees can see their local video on
    * Verify that both attendees can see the other attendee’s remote video
    * Both attendees turn off their video
    * Verify that both attendees’ local videos off
    * Verify that both attendees’ remote videos off


**Testing**

1. Have you successfully run `npm run build:release` locally? yes
2. How did you test these changes? manullly and run the chromDrive
3. Can these changes be tested using the demo application? If yes, which demo application can be used to test it? Video_test
4. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved? No
5. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved? No


By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
